.eslintrc.js

```diff
@@ -13,12 +15,50 @@ module.exports = {
     'react',
     'lingui',
     'simple-import-sort',
+    'bsky-internal',
   ],
   rules: {
     // Temporary until https://github.com/facebook/react-native/pull/43756 gets into a release.
     'prettier/prettier': 0,
     'react/no-unescaped-entities': 0,
     'react-native/no-inline-styles': 0,
+    'bsky-internal/avoid-unwrapped-text': [
+      'error',
+      {
+        impliedTextComponents: [
```

Comment:
The goal is to get this list close to zero, maybe except H1..H6. For components that always return Text (e.g. links), I propose adding Text suffix and making the rule enforce that they actually return Text. Will need follow-up work.